packages/tutanota-app/tutanota-app.pacscript

```diff
@@ -0,0 +1,55 @@
+maintainer="Paul Cosma (saenai) <paul.cosma97@gmail.com>"
+
+pkgname="tutanota"
+name="${pkgname}-app"
+version="1"
+description="Tutanota is the world's most secure email service, easy to use and private by design."
+url="https://mail.tutanota.com/desktop/tutanota-desktop-linux.AppImage"
```

Comment:
So the url gets points to the latest file right? How will the `hash` work then?

Reply:
My hopes are that they don't update too frequently haha

I'll look into alternatives

Reply:
@Henryws, I think we can skip the hash check for this.

Reply:
Yeah

Reply:
Uhm
```
Do you want to view/edit the pacscript [y/N] 
[+] INFO: Sourcing pacscript
[*] WARNING: Package does not contain a hash
[+] INFO: URL exists
[+] INFO: Creating dummy package
[sudo] password for saenai: 
Selecting previously unselected package tutanota-app.
(Reading database ... 296881 files and directories currently installed.)
Preparing to unpack .../tutanota-app-pacstall.deb ...
Unpacking tutanota-app (3.89.25-1) ...
Setting up tutanota-app (3.89.25-1) ...
[+] INFO: Installing dependencies
[+] INFO: Retrieving packages
Initializing download: https://mail.tutanota.com/desktop/tutanota-desktop-linux.AppImage
File size: 106.156 Megabyte(s) (111312438 bytes)
Opening output file tutanota-desktop-linux.AppImage
Starting download

[100%] [.............................................................................] [  71.7MB/s] [00:00]

Downloaded 106.156 Megabyte(s) in 1 second(s). (73398.50 KB/s)
[!] ERROR: Hashes don't match

```

Reply:
That's probably a bug.

Reply:
https://github.com/pacstall/pacstall/issues/458 (PR: https://github.com/pacstall/pacstall/pull/459)